packages/manager/modules/pci/src/projects/project/storages/blocks/add/add.controller.js

```diff
@@ -177,8 +177,15 @@ export default class PciBlockStorageAddController {
   }
 
   onRegionChange() {
+    const { region } = this.storage;
     this.displaySelectedRegion = true;
     this.selectedVolumeAddon = null;
+    const volumeAddonsIds = this.volumesAvailability.plans?.filter(
+      ({ regions }) => regions.find(({ name }) => name === region.name),
+    );
+    this.consumptionVolumeAddons = this.totalConsumptionVolumeAddons.filter(
+      ({ planCode }) => volumeAddonsIds.find(({ code }) => code === planCode),
```

Comment:
**niptick** : `.some` should be more appropriate as you check a boolean value

Reply:
Its done